src/components/Period/Period.js

```diff
@@ -85,6 +85,8 @@ class Period extends React.Component {
 
     changeFormValue(selectValuePath, e.target.value);
 
+    console.log(this.inputRef);
```

Comment:
@ViktorSoroka07 could you please remove `console.log`

Reply:
Sure, good point.